lib/graphql/schema/member/build_type.rb

```diff
@@ -125,6 +125,39 @@ def camelize(string)
           camelized
         end
 
+        # Resolves constant from string (based on Rails `ActiveSupport::Inflector.constantize`)
+        def constantize(string)
+          camel_cased_string = camelize(string)
```

Comment:
I'm not sold we should auto-camelize. I think it's simpler to find a Ruby constant by name. What's the use case for auto-camelizing?

Reply:
I suppose the idea is that it makes it more resilient to errant formatting in the class name (say they pass in `user` instead of `User`), but you're right in that it's unnecessary since that problem should be quickly discovered and fixed on its own.